src/obsidian_tears_frontend/src/pages/game.jsx

```diff
@@ -96,10 +96,11 @@ const Game = (props) => {
       });
       addEventListener(
         "BuyItem",
-        async function (shopIndex, itemIndex, qty, objectName) {
-          // TODO: please remove once Unity has removed it
-          console.log("Frontend - Event BuyItem deprecated");
-        },
+        // Throwed a ESlint error
```

Comment:
Que é isto??? 🤦

Então dá erro e a tua solução é comentar tudo? 😂 
Primeiro o console.log é relevante, pois preciso de saber se a função continua a ser chamada do Unity ou não.
Depois, não achas que este "addEventListener" não exige / espera que haja esse callback? O jogo quando chamar essa função não vai crashar no frontend / web (e ter side effects indesejados)?
Eu não testei, mas acho que é muito provável 😅 
Vou tentar do meu lado ver qual era o erro e ver se consigo resolver 👍

Reply:
Ok, fixed this here: https://github.com/obsidian-tears/obsidian_tears_frontend_backend/pull/52/commits/c5c99da1bfc6fa1e294a6e88f948f40d9c507e7e